db/datastore.py

```diff
@@ -275,3 +275,39 @@ def get_followers(self):
             return None
         else:
             return proto[0]
+
+class MessageStore(object):
+    def __init__(self):
+        self.db = lite.connect(DATABASE)
+        self.db.text_factory = str
+        try:
+            cursor = self.db.cursor()
+            cursor.execute('''CREATE TABLE messages(guid BLOB , signed_pubkey BLOB,
+encryption_pubkey BLOB, subject TEXT, message_type TEXT, message TEXT, timestamp, INTEGER, signature BLOB)''')
+            cursor.execute('''CREATE INDEX idx1 ON messages(guid);''')
+            self.db.commit()
+        except Exception:
+            pass
```

Comment:
it's just a bit weird seing such a defining db operation lightly used on a constructor.
The api user might not be aware, might try to invoke MessageStore on different threads, etc.
I'd def. move that logic to a method that describes that action.

Is there another place where all tables are created? I thought there was some class that happens on the first OB run where all tables are created. Perhaps that's a better place for this, also easier to maintain "where the hell is that table created?" kind of thing going through a maintainer's mind in the future.